### DISCUSSION TITLE
求中央民族大学的相关介绍

### DISCUSSION THREAD

**Discussion by Irokoooo** [CollegesChat/university-information#236] [University]
例如生活条件，专业情况，老师授课如何，住宿条件，保研率与校园生活等，谢谢各位学长学姐

**Comment by Paul996007** [comment]
生活条件不好，住宿条件一坨，丰台5人间爆改8人间，4人间爆改6人间，海淀全是8人间。之前有好几个同学吃基本伙吃得食物中毒拉医院去了都。校园生活如果你在丰台那通勤基本来回4小时以上，海淀倒是方便。
专业开得很多，但强势专业在文科。
